Classes/Search/SearchBarCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SearchBarCell.swift
+//  Freetime
+//
+//  Created by Sherlock, James on 28/07/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+final class SearchBarCell: UICollectionViewCell, UISearchBarDelegate {
+
+    weak var delegate: SearchBarSectionControllerDelegate?
+    private let searchBar = UISearchBar()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
```

Comment:
Just a cell with a search bar, correct - Reason for not putting it directly in the view controller came down to the layout mostly - Couldn't find a nice way (that wasn't causing issues with autolayout) to add the search bar above the collection view/below the nav bar hence it's in a cell.

I don't think this needs more accessibility than the search bar provides already?